src/components/tx/TxData.js

```diff
@@ -287,6 +300,10 @@ class TxData extends React.Component {
       }
     }
 
+    const renderDataScript = (data) => {
+      return `${data} [Data Script]`;
```

Comment:
Maybe just `[Data]`.

Reply:
I agree it's better. Changed here: https://github.com/HathorNetwork/hathor-explorer/pull/141/commits/61751ea06f66ec5e82c084941af18b7fbca506a9